app/config.py

```diff
@@ -164,7 +164,7 @@ class E2ETest(Staging):
 
     # Borrowed from test environment
     TESTING = True
-    WTF_CSRF_ENABLED = False
+    WTF_CSRF_ENABLED = True
```

Comment:
Noting here that this is enabling the CSRF checks in the `end-to-end` test environment, though this is probably correct because we're simulating a user with the end-to-end tests, and I see they still passed here on this PR. 👍